rpm/droid-system-pdx201.spec

```diff
@@ -6,3 +6,11 @@
 %include droid-system-common.inc
 %include droid-system-device/droid-system.inc
 
+%pre
+# This may/should be removed after the next stop release after 4.2.0.
```

Comment:
Is it useful to add a temporary fix to the template?

Reply:
It is better when a known issue is workable and discoverable, so other ports could benefit (this has been raised internally whether it could help [1]).
All modern Xperia ports  that support OTA during 4.2.0 might fall victim to this.

4.3.0 may or may not be the stop release, so we cannot remove this just yet.

Alternatively, this should be documented in HADK FAQ.

[1] https://github.com/sonyxperiadev/bug_tracker/issues/688

Reply:
Any news of this? I think this is still worth while to merge.

Reply:
I've split the PR into byte-size chunks:) https://github.com/mer-hybris/droid-system-sony-template/pull/9

Reply:
4.3.0 and 4.4.0 were stop release, this should not be needed.